src/components/Input/Input.mdx

```diff
@@ -46,8 +46,9 @@ A Input can be used to let the user insert text using key strokes. A crucial for
 ### Limited amount of characters
 
 <Playground>
-  {/* .countryCodeInput { text-align: center; width: 30px; } */}
-  <Input name="countryCode" maxLength={2} className={styles.countryCodeInput} placeholder="XX" />
+  {/* .countryCode { width: 30px; } */}
```

Comment:
I displayed the used CSS code in a code block that highlights CSS syntaxes. I think it looks nicer in the documentation. What do you think about this?

Reply:
Yes, way better!